### PR TITLE
Add Galaxy Chain

### PR DESCRIPTION
**Pull request by jnovinger**
Adds new chain, Galaxy Chain.

Data sourced from [Discord](https://discord.gg/DkPNtpJQ8C), [community explorer](https://explorer.postcapitalist.io/Galaxy/staking/galaxyvaloper1wksgqczx2l5hkqkj5geywcyxrvd0mfqwsmp35s), [Twitter](https://twitter.com/glxuniverse), and [Github](https://github.com/galaxies-labs/networks).

List of genesis peers taken from [here](https://github.com/galaxies-labs/networks/blob/main/peers.md).

~Assuming everything checks out, the only thing is missing is RPC, gRPC, and API entries. Anybody?~ Got RPC/REST node info from the community.

### CODE REVIEW

**Comment by jnovinger** [comment]
> Hi @jnovinger Can you fix the errors? Seems like something's wrong with your chain.json :)

Can do, thanks!

**Comment by jnovinger** [comment]
Bump for @nooomski , fixed that silly error. Sorry, but seems like the workflows don't automatically run after the first one is enabled. :disappointed:

File: galaxy/chain.json [hunk 1]
```diff
@@ -0,0 +1,133 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "galaxy-1",
```
Comment (sunnya97) [inline galaxy/chain.json:3]:
```suggestion
    "chain_name": "galaxy",
```

File: galaxy/chain.json [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    "pretty_name": "Galaxy",
+    "chain_id": "galaxy-1",
+    "bech32_prefix": "galaxy",
+    "daemon_name": "galaxyd",
+    "node_home": "$HOME/.galaxy",
+    "genesis": {
+        "genesis_url": "https://media.githubusercontent.com/media/galaxies-labs/networks/main/galaxy-1/genesis.json"
+    },
+    "codebase": {
+        "git_repo": "git@github.com:galaxies-labs/galaxy.git",
```
Comment (sunnya97) [inline galaxy/chain.json:15]:
```suggestion
        "git_repo": "https://github.com/galaxies-labs/galaxy",
```

File: galaxy/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            "denom": "glx",
+            "exponent": 6
+          }
+        ],
+        "base": "uglx",
+        "name": "Galaxy",
+        "display": "glx",
+        "symbol": "GLX",
+        "logo_URIs": {
+          "svg": "https://raw.githubusercontent.com/galaxies-labs/galaxy-website/main/public/assets/images/logo.svg"
```
Comment (sunnya97) [inline galaxy/assetlist.json:22]: Please move into this repo as a local file in an /images subfolder

**Comment by jnovinger** [comment]
Thanks!

**Pull request merged**